### ISSUE TITLE
Close SidebarLeftFooter menu option after being routed

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#631]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The left sidebar is coming along nicely, but one of the features that's lacking is that we need to close a menu field is the footer if something has been selected. This behavior can be seen if say you're on the [home page](https://activist.org/home/) and then you select `Create` and then `New organization`. You'll be navigated to the organization creation page, but then the `Create` menu will still be open.

This issue would close the menu if something is selected from it. Changes for this would need to be made within [SidebarLeftFooter](https://github.com/activist-org/activist/blob/main/frontend/components/sidebar/left/SidebarLeftFooter.vue) 😊

### Contribution

Happy to support someone with interest in working on this or get to it myself eventually! :)

**Comment by AkshithaReddy1899** [comment]
Hello @andrewtavis I would like to work on this issue, can you please assign this to me.

Thank you.

**Comment by andrewtavis** [comment]
Hi @AkshithaReddy1899 👋 Thanks for your interest in working with us! I already had a conversation with @krallowski about working on this. Let's wait for a confirmation on whether they'd like to work on it still, but you'd be very welcome to take it over if not 😊

**Comment by andrewtavis** [comment]
We're still looking for someone to work on #369 where the captcha component would be finalized if that would be of interest! The issue has lots of documentation already. Not the easiest issue, but happy to discuss or please check back for another issue later as there will be many! :)

**Comment by krallowski** [comment]
Heyy :)
Yeah, I can take it up, but it would take some time as the following days are quite packed. So if you have more spare time to work on it I do not want to hold you back. @AkshithaReddy1899 if you want to take this up just assign it to you, otherwise you can assign that to e if it is okay @andrewtavis :)
BR

**Comment by andrewtavis** [comment]
Let me assign this to you @krallowski as we have discussed it before :) Sorry this wasn't in the issue, @AkshithaReddy1899! We were trying to decide which issue would be best. Please let me know if something else is of interest, and also happy to let you know if there's a new one!

**Comment by andrewtavis** [comment]
Hey @krallowski :) Following up here for documentation as you suggested 😊

So the sidebar opening and closing on hover is intended, and that the menu stays open when we hover, open it, go away such that it collapses and then hover again. What we're looking at here specifically is that when we click an option like `Creat` > `New organization` and go to the new page, then the sidebar options are still open. We do want the menu options to close in this case :) 

Let me know if this makes sense, and thanks for reaching out!

**Comment by krallowski** [comment]
Hi @andrewtavis :) Okay, I got it. Thanks for the clarification :)

**Comment by andrewtavis** [comment]
No stress, @krallowski! Happy to help :)

**Comment by krallowski** [comment]
To keep accessbility in mind it was decided to set the focus to the first breadcrumb of the new loaded page or on the PageSubPageSelector in case there is one.

**Comment by tatianakharitonovich** [comment]
Hi there!
While I was trying to deal with this issue I noticed there was the same code three times in [SidebarLeftFooter](https://github.com/activist-org/activist/blob/main/frontend/components/sidebar/left/SidebarLeftFooter.vue).

I supose it is not good and refactored it.
My cahnges in PR https://github.com/activist-org/activist/pull/656.
I hope it is usefull!

Thank you!

**Comment by andrewtavis** [comment]
Hey @tatianakharitonovich 👋 Thanks for your interest in helping out :) Does the code you sent along also solve this issue here, or is it just a refactor? @krallowski, in case it's a refactor, I can check and then you can still send along the change :)

**Comment by tatianakharitonovich** [comment]
Hi! It is just a refactor!

**Comment by andrewtavis** [comment]
Looking into this in conversation with @krallowski, I believe that #702 will close this, so looks like @tatianakharitonovich will take this one down after all 😊 Marking this as closed once the PR is merged as testing in the deploy preview indicates that it will function properly 🥳
